pkg/components/components.go

```diff
@@ -1146,7 +1146,6 @@ func GetCSVBase(params *CSVBaseParams) *csvv1alpha1.ClusterServiceVersion {
 			Description: params.Description,
 			Keywords:    []string{"KubeVirt", "Virtualization"},
 			Version:     csvVersion.OperatorVersion{Version: params.Version},
-			Replaces:    params.Replaces,
```

Comment:
If this is the only place that uses `params.Replaces` please remove it completely - from the struct and from the command line arguments

Reply:
I changed it, and the removal of `replaces` field in the CSV occurs only on the workflow